src/Umbraco.Web.UI.Client/src/views/content/content.sendToTranslate.controller.js

```diff
@@ -0,0 +1,67 @@
+(function () {
+    "use strict";
+
+    function ContentSendToTranslateController($scope, $q, usersResource, localizationResource, contentResource) {
```

Comment:
currently the convention for angular files is to have the file names all lowercase